test/parallel/test-http-hightwatermark.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const net = require('net');
+const http = require('http');
+
```

Comment:
Can you add a description of the test here?

Reply:
Thank you for your review.
I added a description. Please check it again.